libs/common-services/src/test/java/uk/gov/di/ipv/core/library/fixtures/VcFixtures.java

```diff
@@ -522,6 +523,43 @@ static VerifiableCredential vcExperianFraudScoreOne() {
                 Instant.ofEpochSecond(1704290386));
     }
 
+    static VerifiableCredential vcFraudExpired() {
+        TestVc.TestCredentialSubject credentialSubject =
+                TestVc.TestCredentialSubject.builder()
+                        .address(List.of(ADDRESS_3))
+                        .birthDate(List.of(new BirthDate("1959-08-23")))
+                        .build();
+        return generateVerifiableCredential(
+                TEST_SUBJECT,
+                EXPERIAN_KBV_CRI,
+                TestVc.builder()
+                        .evidence(FRAUD_EVIDENCE_NO_CHECK_DETAILS)
+                        .credentialSubject(credentialSubject)
+                        .build(),
+                "https://review-f.integration.account.gov.uk",
+                Instant.ofEpochSecond(1658829758));
```

Comment:
Is it possible to pass through a clock into this builder, so we can guarantee that we always have a valid/expired VC for these tests? 

Then we may be able to get around having to set the expiry duration to something massive as well